components/clarinet-sdk/node/vitest-helpers/src/clarityValuesMatchers.ts

```diff
@@ -286,7 +290,7 @@ expect.extend({
       return errorToAssertionResult.call(this, e);
     }
 
-    const actualString = principalToString(actual);
+    const actualString = cvToString(actual, "tryAscii");
```

Comment:
Why not the `Cl.stringify` aka `prettyPrint` that you added?

Reply:
If you think this is the best replacement for `principalToString` i should use it

Reply:
In this particular case, we don't want the `'` at the beginning of the string. 
But Cl.stringify rightfully puts it.
I'm tempting to stick to cvToString, unless you tell me it'll be deprecated in the future

Reply:
```suggestion
    const actualString = Cl.prettyPrint(actual, 2).replace(/^'/, "");
```

Reply:
Got it, that makes sense. No no, fine to use either. Was just curious :)